### PR TITLE
Test working app

### PR DESCRIPTION
**Pull request by panpilkarz**
This PR fix test flow and add a new test that does:

invoke runmultiple with click runner,
creates a name reg contract
sends transactions to name reg contract
validates:
contract creation
new blocks are mined
contract processes transactions and updates as expected

### CODE REVIEW

File: hydrachain/tests/test_working_app.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+        gevent.sleep(15)
+        for app in apps:
+            app.stop()
+
+    app.serve_until_stopped = mock_serve_until_stopped
+    runner = CliRunner()
+    with runner.isolated_filesystem():
+        datadir = 'datadir{}'.format(gasprice)
+        runner.invoke(app.pyethapp_app.app, ['-d', datadir, 'runmultiple'])
+        #runner.invoke(app.pyethapp_app.app, ['-d', datadir,
```
Comment (konradkonrad) [inline hydrachain/tests/test_working_app.py:157]: Any reason you want to keep the commented code here?

Comment (panpilkarz) [inline hydrachain/tests/test_working_app.py:157]: It is helpful when you need to trouble shoot as it turns on debugging.

File: hydrachain/tests/test_working_app.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+            assert receipt['blockHash'] == tx['blockHash']
+            assert receipt['blockHash'] == recent_block['hash']
+
+            # Get contract address from receipt
+            contract_address = receipt['contractAddress']
+            code = self.client.call('eth_getCode', contract_address)
+            self.log('eth_getCode OK {}'.format(code))
+
+            assert code.startswith('0x')
+            assert len(code) > len('0x')
```
Comment (konradkonrad) [inline hydrachain/tests/test_working_app.py:116]: Couldn't we tighten this assertion a little bit? `code` could be almost anything... ;)

Comment (panpilkarz) [inline hydrachain/tests/test_working_app.py:116]: Was discussed with Heiko. That is enough. Conversation is on Slack.

**Pull request merged**